src/Internal/Service/Level3Service.php

```diff
@@ -0,0 +1,192 @@
+<?php
+/**
+ * Class Level3Service
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Internal\Service;
+
+use stdClass;
+use WC_Order_Item;
+use WC_Order_Item_Product;
+use WC_Order_Item_Fee;
+use WC_Payments_Account;
+use WC_Payments_Utils;
+use WCPay\Internal\Service\OrderService;
+use WCPay\Exceptions\Order_Not_Found_Exception;
+
+/**
+ * Service for generating Level 3 data from orders.
+ */
+class Level3Service {
+	/**
+	 * Order service.
+	 *
+	 * @var OrderService
+	 */
+	private $order_service;
```

Comment:
Since we are using PHP 7.4 now, can we maybe type hint `OrderService` here (and in similar places)? @htdat can you perhaps pull the data for us and tell how many merchants are using WooCommerce Payments for PHP version below 7.4?

Reply:
Considering that we'd still need PHPDoc comments (unless we change our PHPCS rules), and IDEs understand them well, is this worth doing for now? With Psalm in place, we cannot "hack" this anyway, we need the right types.

Reply:
While it's not a priority right now, we could consider adding it later as a minor enhancement or during a hack week. I believe it's not necessary to include it in this PR or even this project phase.

Reply:
>  @htdat can you perhaps pull the data for us and tell how many merchants are using WooCommerce Payments for PHP version below 7.4?

See here 32027-pb/#bash

As long as WooPayments still supports PHP 7.3, we should not use the new syntax. But we can consider doing that duing a HACK week or a cooldown week.